requirements.txt

```diff
@@ -1,7 +1,7 @@
 # setup.py install_requires
 # any changes here should also be reflected in setup.py "install_requires"
 aws-embedded-metrics == 3.2.0
-boto3 ~= 1.20
+boto3 ~= 1.34
 getdaft==0.2.23
```

Comment:
Do you want to upgrade daft to `0.2.25` too?

Reply:
Will do.

Reply:
Upgraded daft.